public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/appInsights.json

```diff
@@ -300,18 +297,22 @@
           {
             "matcher": {
               "id": "byName",
-              "options": "Server requests"
+              "options": "users/count_unique"
             },
             "properties": [
               {
                 "id": "links",
                 "value": [
                   {
                     "targetBlank": true,
-                    "title": "",
-                    "url": "https://ms.portal.azure.com/#blade/Microsoft_Azure_MonitoringMetrics/Metrics.ReactView/ResourceId/%2Fsubscriptions%2F${sub}%2FresourceGroups%2F${rg}%2Fproviders%2Fmicrosoft.insights%2Fcomponents%2F${res}/TimeContext/%7B%22relative%22%3A%7B%22duration%22%3A86400000%7D%2C%22options%22%3A%7B%22grain%22%3A1%2C%22showUTCTime%22%3Afalse%7D%7D/Chart/%7B%22metrics%22%3A%5B%7B%22resourceMetadata%22%3A%7B%22id%22%3A%22%2Fsubscriptions%2F${sub}%2FresourceGroups%2F${rg}%2Fproviders%2Fmicrosoft.insights%2Fcomponents%2F${res}%22%7D%2C%22name%22%3A%22requests%2Fcount%22%2C%22namespace%22%3A%22microsoft.insights%2Fcomponents%22%2C%22metricVisualization%22%3A%7B%22displayName%22%3A%22Server%20requests%22%7D%2C%22aggregationType%22%3A7%2C%22thresholds%22%3A%5B%5D%7D%5D%2C%22filterCollection%22%3A%7B%22filters%22%3A%5B%5D%7D%2C%22grouping%22%3Anull%2C%22visualization%22%3A%7B%22chartType%22%3A2%2C%22legendVisualization%22%3A%7B%22isVisible%22%3Atrue%2C%22position%22%3A2%2C%22hideSubtitle%22%3Afalse%7D%2C%22axisVisualization%22%3A%7B%22x%22%3A%7B%22isVisible%22%3Atrue%2C%22axisType%22%3A2%7D%2C%22y%22%3A%7B%22isVisible%22%3Atrue%2C%22axisType%22%3A1%7D%7D%2C%22disablePinning%22%3Atrue%7D%2C%22title%22%3A%22Count%20Server%20requests%20for%20${res}%22%2C%22titleKind%22%3A1%2C%22timespan%22%3A%7B%22relative%22%3A%7B%22duration%22%3A86400000%7D%2C%22showUTCTime%22%3Afalse%2C%22grain%22%3A1%7D%2C%22ariaLabel%22%3Anull%7D/openInEditMode/true"
+                    "title": "${res}  | Users",
+                    "url": "https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/${sub}/resourceGroups/${rg}/providers/microsoft.insights/components/${res}/segmentationUsers"
```

Comment:
If I remember correctly, from getting it in other occasions, that is not an error on the link but rather on the panel. It seems like it is for the link because the error shows in the same place as the panel. Could you share a screenshot of the variables that you are using in the dropdowns?

Reply:
When I click on the link, I get a 401 from the link in Azure Portal, but I think it's trying to connect me to the wrong account and won't let me switch or something. It's probably more to do with my set up or a bug with Azure than it is an issue with this link though, so maybe we can just ignore it for now

<img width="816" alt="Screen Shot 2022-03-03 at 2 29 50 PM" src="https://user-images.githubusercontent.com/6620164/156638746-8416530a-a36b-4a30-8699-cefc3685dc16.png">

Reply:
I just tried the link again and do not encounter any issues. Is `microsoft.insights/components/AppInsightsTestData` the resource you used to test the dashboard? If so, what happens when you try to access the resource directly in the portal?